docs/sources/developers/plugins/package-a-plugin.md

```diff
@@ -0,0 +1,65 @@
+---
+description: How to package a plugin
+title: Package a plugin
+---
+
+You've just built your first plugin and you're getting ready to [publish it](publish-a-plugin.md), and now you need to package it. This step follows the building and [signing](sign-a-plugin.md) steps for plugin deployment. 
```

Comment:
Shouldn't `relref` be used here or isn't it needed when in the same directory? There are some other similar changes where this is applicable. Ref https://grafana.com/docs/writers-toolkit/writing-guide/references/

Reply:
It may be, I will double-check and update/fix rel-refs.

Reply:
Note: latest advice is in https://github.com/grafana/writers-toolkit/pull/225.

Reply:
@jdbaldry Good to know. I probably missed it, but I've looked in the writer's toolkit and don't see any explicit examples of links to files in the same directory. Chris confirmed that relrefs are needed there, so I've added them in.

Reply:
LGTM, thanks!